src/uk/gov/hmcts/contino/GradleBuilder.groovy

```diff
@@ -256,6 +256,13 @@ EOF
     return gradleWithOutput("buildEnvironment").contains(pluginName)
   }
 
+  @Override
+  def securityScan() {
+    localSteps.writeFile(file: 'security.sh', text: localSteps.libraryResource('uk/gov/hmcts/pipeline/security/security-backend/security.sh'))
+    this.securitytest.execute()
+    WarningCollector.addPipelineWarning("security.sh_moved", "Please remove security.sh from root of repository, no longer needed as it has been moved to the Jenkins library", LocalDate.of(2023, 03, 29))
```

Comment:
this needs to check if the file exists in the repository not just add unconditionally, you can check with `fileExists(..)`